tartufo/util.py

```diff
@@ -50,13 +48,6 @@ def del_rw(_func: Callable, name: str, _exc: Exception) -> None:
     os.remove(name)
 
 
-def convert_regexes_to_rules(regexes: Dict[str, Pattern]) -> Dict[str, Rule]:
-    return {
-        name: Rule(name=name, pattern=pattern, path_pattern=None, re_match_type="match")
-        for name, pattern in regexes.items()
-    }
```

Comment:
Are you certain we don't still need this? Isn't this how the `default_regexes.json` gets loaded? https://github.com/godaddy/tartufo/blob/main/tartufo/data/default_regexes.json

Reply:
Yes, `convert_regexes_to_rules` has 0 references. `default_regexes.json` gets loaded in `configure_regexes`